SECURITY.md

```diff
@@ -49,16 +88,18 @@ LoopBack implements the
 
 </details>
 
-Within LoopBack 4, fixes are not backported across semver major versions of a
-package.
+**Within LoopBack 4, fixes are not backported across semver major versions of a
+package.**
+
+We don't provide any
+[LTS version for LoopBack 4](https://github.com/strongloop/loopback-next/issues/4398)
+yet.
 
-## Security advisories
+## Security Advisories
 
 Security advisories can be found on the
 [LoopBack website](https://loopback.io/doc/en/sec/index.html).
 
-## Reporting a vulnerability
-
-If you think you have discovered a new security issue with any StrongLoop
-package, **please do not report it on GitHub**. Instead, send an email to
-reachsl@us.ibm.com with a full description and steps to reproduce.
+These advisories have not been published to the
+[Node.js vulnerability database](https://github.com/nodejs/security-wg/blob/0bf201409d659e0cca19fc32815b5dabfde14406/processes/vuln_db.md)
```

Comment:
Should we perhaps cross publish those existing advisories to Node.js vulnerability database too?

Reply:
I think we should publish them, as per-#4781's 3rd suggestion